src/many-ledger/src/storage.rs

```diff
@@ -97,8 +105,9 @@ impl LedgerStorage {
         persistent_path: P,
         blockchain: bool,
     ) -> Result<Self, String> {
-        let mut persistent_store = fmerk::Merk::open(persistent_path).map_err(|e| e.to_string())?;
+        let snapshot_path = persistent_path.as_ref().parent().unwrap().to_path_buf();
```

Comment:
`snapshot_path` should come from a command-line flag (with this being a good default if the flag is missing). We will want an empty directory for snapshots.